src/token-manager/index.ts

```diff
@@ -18,7 +18,7 @@ import { parseQuery } from '../util/url';
 
 interface GetTokensOptions extends ConfigOptions {
   forceRenew?: boolean;
-  login?: 'native' | 'redirect' | undefined;
+  login?: 'embedded' | 'redirect' | undefined;
```

Comment:
Only code change required. Since there are only two choices, the generic `else` in this condition is what triggers the embedded functionality: https://github.com/ForgeRock/forgerock-javascript-sdk/pull/100/files#diff-8c14601248b947426b7c11ac7d595b651088ef97e5dc330af90524d20af79b6bR128